src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+ width: $calendar-width;
+ display: flex;
+ flex-wrap: wrap;
+ gap: $gap-between;
+ margin: 20px;
+
+ &__day {
+  position: relative;
+  display: flex;
+  justify-content: center;
+
+  width: $day-size;
+  height: $day-size;
+  line-height: $day-size;
+
+  border: $day-border;
+  color: $day-text-color;
+  font-family: $day-font;
+  font-size: $day-text-size;
+  background-color: $day-background-color;
+
+  &:hover {
+    cursor: pointer;
+    background-color: $hover-color-day;
+    transform: translateY(-20px);
+    transition: transform 0.5s ease-in-out;
```

Comment:
Move it to the `__day` class,bc if you put it to the hover - you don't have unhover transition
```suggestion
```